ambassador/ambassador/envoy/v2/v2listener.py

```diff
@@ -27,48 +27,24 @@
 if TYPE_CHECKING:
     from . import V2Config
 
-
-ExtAuthRequestHeaders = {
-    'Authorization': True,
-    'Cookie': True,
-    'Forwarded': True,
-    'From': True,
-    'Host': True,
-    'Proxy-Authenticate': True,
-    'Proxy-Authorization': True,
-    'Set-Cookie': True,
-    'User-Agent': True,
-    'X-Forwarded-For': True,
-    'X-Forwarded-Host': True,
-    'X-Forwarded-Proto'
-    'X-Gateway-Proto': True,
-    'WWW-Authenticate': True,
-}
-
 @multi
 def v2filter(irfilter):
     return irfilter.kind
 
 @v2filter.when("IRAuth")
 def v2filter(auth):
-    request_headers = dict(ExtAuthRequestHeaders)
-
-    for hdr in auth.allowed_headers:
-        request_headers[hdr] = True
-
     return {
         'name': 'envoy.ext_authz',
         'config': {
             'http_service': {
                 'server_uri': {
                     'uri': 'http://%s' % auth.auth_service,
                     'cluster': auth.cluster.name,
-                    'timeout': '3s',
+                    'timeout': auth.timeout_ms,
```

Comment:
What happens with `auth.timeout_ms` for a v0 `AuthService` instead of a v1 `AuthService`?

Reply:
Well, if we are going to support V0, I assume that both cases should be supported. What we can't do IMO, is to hard-code `3s` and then disable user-input as it was before.

Reply:
👍